src/ui/control/logo_control.js

```diff
@@ -21,9 +21,14 @@ class LogoControl {
     onAdd(map) {
         this._map = map;
         this._container = DOM.create('div', 'mapboxgl-ctrl');
+        const anchor = DOM.create('a', 'mapboxgl-ctrl-logo');
+        anchor.target = "_blank";
+        anchor.href = "https://www.mapbox.com/";
+        anchor.setAttribute("aria-label", "Mapbox logo");
+        this._container.appendChild(anchor);
+        this._container.style.display = 'none';
 
         this._map.on('sourcedata', this._updateLogo);
-        this._updateLogo();
```

Comment:
Sure! I actually had that for a second, but after I moved the `DOM.create` stuff to `onAdd` I didn't think it was necessary anymore. But there is a potential race condition if  source fires its `metadata` event before the LogoControl is added 👍